sources/getRole.ts

```diff
@@ -14,67 +15,118 @@ export function getLocalName(element: Element): string {
 }
 
 const localNameToRoleMappings: Record<string, string | undefined> = {
+	address: "group",
 	article: "article",
+	// WARNING: Only in certain context
 	aside: "complementary",
+	b: "generic",
+	bdi: "generic",
+	bdo: "generic",
+	blockquote: "blockquote",
+	body: "generic",
 	button: "button",
+	caption: "caption",
+	code: "code",
+	data: "generic",
 	datalist: "listbox",
+	// WARNING: html-aria and html-aam conflict on this role assignment
 	dd: "definition",
+	del: "deletion",
 	details: "group",
+	dfn: "term",
 	dialog: "dialog",
+	div: "generic",
+	// WARNING: html-aria and html-aam conflict on this role assignment
 	dt: "term",
+	em: "emphasis",
 	fieldset: "group",
 	figure: "figure",
+	// WARNING: Only in certain context
+	footer: "contentinfo",
 	// WARNING: Only with an accessible name
 	form: "form",
-	footer: "contentinfo",
 	h1: "heading",
 	h2: "heading",
 	h3: "heading",
 	h4: "heading",
 	h5: "heading",
 	h6: "heading",
+	// WARNING: Only in certain context
 	header: "banner",
+	// WARNING: html-aria and html-aam conflict on this role assignment
+	hgroup: "group",
 	hr: "separator",
 	html: "document",
+	i: "generic",
+	ins: "insertion",
 	legend: "legend",
+	// WARNING: Only in certain context
 	li: "listitem",
-	math: "math",
 	main: "main",
+	math: "math",
 	menu: "list",
+	meter: "meter",
 	nav: "navigation",
 	ol: "list",
 	optgroup: "group",
 	// WARNING: Only in certain context
 	option: "option",
 	output: "status",
+	p: "paragraph",
+	pre: "generic",
 	progress: "progressbar",
+	q: "generic",
+	// WARNING: html-aria and html-aam conflict on this role assignment
+	s: "deletion",
```

Comment:
TODO: ref https://github.com/w3c/html-aam/issues/416 vs mapping in https://w3c.github.io/html-aria/#docconformance (no issue yet) (https://github.com/w3c/html-aria/blob/4d9fc05ef1acdfd63944e63631c10f449e68c8e5/index.html#L2616)

Reply:
Raised https://github.com/w3c/html-aria/issues/466